.github/workflows/web-test.yaml

```
@@ -0,0 +1,100 @@
+      - name: Start HTTP server for test pages
+        run: |
+          python3 -m http.server 8080 --directory sample-test/src/main/resources/web-pages &
+          echo $! > server.pid
+          sleep 2
+
+      - name: Setup Chrome and ChromeDriver
+        run: |
+          # Download latest stable ChromeDriver for Linux
+          LATEST_VERSION=$(curl -s "https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions.json" | jq -r '.channels.Stable.version')
+          echo "Using ChromeDriver version: $LATEST_VERSION"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)